CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased]
 ------------
+### Added
+
+- Ability to debug ELF enclaves on Windows using
```

Comment:
@anakrish Thanks for updating CHANGELOG

Reply:
Done. Available here: https://github.com/openenclave/openenclave/blob/anakrish/windows-debugging-docs/docs/GettingStartedDocs/Windows_windbg.md